### PR TITLE
[9.0] [MIG] account_due_list_days_overdue

### PR DESCRIPTION
**Pull request by JordiBForgeFlow**
PR depends on https://github.com/OCA/account-payment/pull/122

### CODE REVIEW

**Comment by JordiBForgeFlow** [comment]
cc @darshan-serpent @eficent

**Comment by serpentcs-dev1** [comment]
LGTM :+1: 

Still need to migrate the test cases. Will do the needful.

File: account_due_list_days_overdue/models/account_overdue_term.py [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+    @api.multi
+    @api.depends('from_day', 'to_day')
+    def _compute_technical_name(self):
+        for rec in self:
+            rec.tech_name = 'overdue_term_%d_%d' % (rec.from_day, rec.to_day)
+
+    @api.model
+    def create(self, vals):
+        res = super(AccountDaysOverdue, self).create(vals)
+        if self.pool['account.move.line']._register_hook(self.env.cr):
```
Comment (LoisRForgeFlow) [inline account_due_list_days_overdue/models/account_overdue_term.py:31]: why are you calling this with self.pool? cannot you use the new self.env?

**Review by LoisRForgeFlow — APPROVED** [review]
Just a question and a minor fix in xml files (remove \<data\> tags and renaming \<openerp\> to \<odoo\>). Anyway it LGTM :+1:

**Review by AaronHForgeFlow — APPROVED** [review]
Tested on local with https://github.com/OCA/account-payment/pull/122

:+1: 

The feature of adding columns in the list view is amazing

File: account_due_list_days_overdue/models/account_overdue_term.py (hunk 1 above)
Comment (JordiBForgeFlow) [reply account_due_list_days_overdue/models/account_overdue_term.py:31]: Beause the method is only avaioable in old api.

**Review by serpentcs-dev1 — APPROVED** [review]
Tested on local, code LGTM :+1: 

Yes, the feature is really helpful.

**Pull request merged**